ab7/frontend-boilerplate
# [ISSUE-14] Bump all dependencies to latest major versions

### Description

Bump all dependencies to latest major versions.

### Ticket

https://github.com/ab7/frontend-boilerplate/issues/14

### Testing

* Pull branch and install dependencies:

    ```
    yarn install
    ```

* Start dev server and verify test page successfully opens up:

    ```
    yarn start
    ```
